clash/the_hunt.tex

```diff
@@ -280,24 +288,28 @@ \subsection*{\MakeUppercase{Additional Rules}}
   \centering
   \medskip
 
-  \newcommand{\bronze}[0]{\svgunit[12]{bronze}}
-  \newcommand{\silver}[0]{\svgunit[12]{silver}}
-  \newcommand{\golden}[0]{\svgunit[12]{golden}}
-  \newcommand{\azure}[0]{\svgunit[12]{azure}}
+  \let\origbronze\bronze
```

Comment:
Didn't you do that above already?

Reply:
The scope of  these redefinitions is per table. I don't know, I like them contained like they are now.